chapters/simple_exec.md

```diff
@@ -1,9 +1,13 @@
 # Built-in Simple Executors and Reserved `uses` in Jina
 
+- [Built-in Simple Executors and Reserved keyword `uses` in Jina](#built-in-simple-executors-and-reserved--uses--in-jina)
+  * [What is Simple Executor?](#what-is-simple-executor)
+  * [What are the reserved `uses`?](#what-are-the-reserved-uses)
+  * [How to use built-in Simple Executors?](#how-to-use-built-in-simple-executors)
 
 ## What is Simple Executor?
 
-In Jina we have several built-in some *simple* executors. They are called simple for two reasons:
+In Jina we have several built-in some *simple* executors, which are located in `jina/resources/`. They are called simple for two reasons:
 
 - It only requires a YAML config, no Python code is required;
```

Comment:
They only require a YAML configuration file. 
They inherit from `BaseExecutor` directly, therefore the logic thus fully relies on the drivers.